CRM/Contact/BAO/Group.php

```diff
@@ -448,9 +443,6 @@ public static function create(&$params) {
         }
       }
 
-      // clear any descendant groups cache if exists
-      $finalGroups = CRM_Core_BAO_Cache::deleteGroup('descendant groups for an org');
```

Comment:
@seamuslee001 aha - it was the removal of this line causing all the issues - it looked pretty innoculous - removing a pointless delete but it was doing a cache clear as a hidden side-effect